bringup/CMakeLists.txt

```diff
@@ -0,0 +1,31 @@
+# ###############################################################################
+# Set minimum required version of cmake, project name and compile options
+# ###############################################################################
```

Comment:
```suggestion
```
This code being commented is idiomatic and straightforward so there is no need for a comment. It's comparable to the classic 
```cpp
// set i to 1
int i = 1;
```